parsec/backend/postgresql/organization.py

```diff
@@ -335,6 +398,56 @@ async def stats(self, id: OrganizationID) -> OrganizationStats:
             users_per_profile_detail=users_per_profile_detail,
         )
 
+    async def server_stats(self, from_date: DateTime, to_date: DateTime):
+        async with self.dbh.pool.acquire() as conn, conn.transaction():
+            query_lines = await conn.fetch(
+                *_q_get_server_stats(
+                    from_date=PyDateTime(
```

Comment:
We have added `parsec._parsec.DateTime` is support to asyncpg (see what is done in `parsec/backend/postgresql/handler.py` so no need to use Python's `datetime.datetime`